Analysis/ITKIsoWavelets.py

```diff
@@ -22,27 +22,29 @@
 fftFilter.Update()
 ComplexType=itk.output(fftFilter.GetOutput())
 RealImageType = itk.Image[itk.F,3]
+inverseFFT = InverseFFTType = itk.InverseFFTImageFilter[ ComplexType, RealImageType].New()
```

Comment:
InverseFFTType does not seem to be used anywhere in the file. It is not necessary to define it.